doc/rest.json

```diff
@@ -367,6 +367,25 @@
                     }
                 }
             }
+        },
+        "/taoLti/KeyPairGenerator/generate": {
```

Comment:
Can we use `keyChain`, instead? It is already called keychain all over the places. We are generation a new nomenclature.

Reply:
PlatformKeyChainRepositoryInterface  is not an interface of PlatformJwksRepository